metrics/web/test/project_groups/domain/usecases/params/project_group_delete_param_test.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:metrics/project_groups/domain/usecases/parameters/delete_project_group_param.dart';
+import 'package:test/test.dart';
+
+import '../../../../test_utils/matcher_util.dart';
+
+void main() {
+  group("ProjectGroupDeleteParam", () {
```

Comment:
Should we test that we can create this class if we specify the `projectGroupId ` param?